packages/video_player/video_player/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Updates minimum Flutter version to 2.10.
 
+## 2.3.1
```

Comment:
The `NEXT` section needs to be included here; see the `publishable` CI failure for details.